src/Api.Rest/Common/Client/RestClientBase.cs

```diff
@@ -466,11 +468,19 @@ private void BuildHttpClient( TimeSpan? timeout )
 				//     i.e. the Server is already authenticating every single request
 				UnsafeAuthenticatedConnectionSharing = true
 			};
+#elif NET

```

Comment:
Sollten wir hier vielleicht generell auf Blazor testen?
`System.OperatingSystem.IsBrowser()`

Reply:
That's possible. AFAIK, Blazor didn't support the HttpClientHandler at all, until recently. Maybe, the generic checks will return true some day. I check specifically for the definition of NET, since only NET contains the attributes and OS classes.